apollo-mockserver/src/main/java/com/ctrip/framework/apollo/mockserver/EmbeddedApollo.java

```diff
@@ -68,8 +67,6 @@ public MockResponse dispatch(RecordedRequest request) throws InterruptedExceptio
         } else if (request.getPath().startsWith("/configs")) {
           List<String> pathSegments = request.getRequestUrl().pathSegments();
           // appId and cluster might be used in the future
-          String appId = pathSegments.get(1);
```

Comment:
how about keeping it?

Reply:
I would also prefer keeping it, as it might be hard future maintainers to understand how to retrieve the appId and cluster

Reply:
  I missed the comment above the codes

Reply:
Reverted back